.github/CODEOWNERS

```diff
@@ -0,0 +1,76 @@
+# Data
+/packages/api-fetch                             @youknowriad @gziolo @aduth @nerrad
```

Comment:
@kadamwhite @mmtr - I know that you both had very meaningful contributions in this package. Would you mind being listed as reviewers for any future code changes in this folder (or any other listed in this config file)?

Reply:
That sounds good to me, thanks @gziolo!